test/Ocelot.UnitTests/DownstreamUrlCreator/DownstreamUrlCreatorMiddlewareTests.cs

```diff
@@ -543,6 +543,70 @@ public void should_fix_issue_748(string upstreamTemplate, string downstreamTempl
                 .BDDfy();
         }
 
+        [Fact]
+        public void Should_map_when_query_parameters_has_same_names_with_placeholder()
+        {
+            var methods = new List<string> { "Post", "Get" };
+            var downstreamRoute = new DownstreamRouteBuilder()
+                .WithUpstreamPathTemplate(new UpstreamPathTemplateBuilder()
+                    .WithOriginalValue("/users?userId={userId}&personId={personId}").Build())
+                .WithDownstreamPathTemplate("/persons?personId={personId}&userId={userId}")
+                .WithUpstreamHttpMethod(methods)
+                .WithDownstreamScheme(Uri.UriSchemeHttp)
+                .Build();
+            var config = new ServiceProviderConfigurationBuilder().Build();
+
+            this.Given(x => x.GivenTheDownStreamRouteIs(
+                    new DownstreamRouteHolder(
+                        new List<PlaceholderNameAndValue>
+                        {
+                            new("{userId}", "webley"),
+                            new("{personId}", "12345"),
```

Comment:
### Again! :grimacing: 
This is copied user scenario from old tests!
**Where is testing of your user scenario :question:**